javascript/javascript2/week2/workingWithMovies.js

```diff
@@ -0,0 +1,28 @@
+//Create an array of movies containing the movies with a short title
+const shortTitleMovies = movies.filter (movie => movie.title.length < 3)
+
+//Create an array of movie titles with long movie titles
+const longTitles = movies.filter (movie => movie.title.length > 50).map (title => title.title)
+
+//Count the number of movies made between 1980-1989 (including both the years)
+const moviesCount = movies.filter (movie => (movie.year >= 1980 && movie.year <= 1989)).length
+
+//Create a new array that has an extra key called tag.
+const moviesWithTag = movies
+moviesWithTag.forEach (movie => (movie.tag = movie.rating >= 7 ? "Good" : (movie.rating >= 4 ? "Average" : "Bad")))
```

Comment:
Chaining ternary operators like this makes code hard to read.
I'd make this longer:
```suggestion
moviesWithTag.forEach(movie => {
  if(movie.rating >= 7) {
    movie.tag = "Good"
  } else if(movie.rating >= 4) {
    movie.tag = "Average"
  } else {
    movie.tag = "Bad"
  }
})
```